test/models/department_test.rb

```diff
@@ -2,13 +2,12 @@
 
 class DepartmentTest < ActiveSupport::TestCase
   def setup
-    @department = Department.new
+    @department = create :department
   end
 
-  test 'department is saveable with user relation' do
-    refute @department.valid?
-    @department.contact = build :contact
+  test 'department is valid' do
     assert @department.valid?
-    assert @department.save!
+    @department.contact.name = ''
```

Comment:
shouldn't this and the following line be in a separate test, like test 'department without required fields is invalid'?